source/_components/isy994.markdown

```diff
@@ -82,6 +84,10 @@ tls:
   description: This entry should reflect the version of TLS that the ISY controller is using for HTTPS encryption. This value can be either 1.1 or 1.2. If this value is not set, it is assumed to be version 1.1. This is the default for most users. ISY994 Pro users may likely be using 1.2. When using HTTPS in the host entry, it is best practice to set this value.
   required: false
   type: string
+isy_variables:
+  description: ISY Variables can be explicitly defined as as sensors, binary sensors, or switches.  See [Variables as Devices](#variables-as-devices) below.
+  required: false
+  type: dict
```

Comment:
`dict` is not a valid configuration type for the documentation.

Reply:
changed to (map) as per https://www.home-assistant.io/components/sensor.mqtt/ `device`